content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -422,14 +422,14 @@ An `int` option requires a range attribute to specify the minimum and maximum ra
      
     [[range(3, 16)]] // This integer option accepts values between 3 and 16 (both ends included).  
     option int o_numberOfTaps;
-  
-Even though the Shader Variant Options are declared globally, they are actually encoded in a single array of bits as a member variable of one, and only one, SRG.  
-To learn more about how Shader Variant Options are encoded when compiled, please read: [Shader Variant Options & The Fallback Key.](shader-variants-fallback-key)  
+```
+Even though the Shader variant options are declared globally, they are actually encoded in a single array of bits as a member variable of one, and only one, SRG.  
+To learn more about how Shader variant options are encoded when compiled, please read: [Shader variant options & The Fallback Key.](shader-variants-fallback-key)  
   
 # <a name="padton"></a>The special attribute [[pad_to(N)]]
 In DX12, the layout, offset & sizes, of variables inside `struct` definitions changes whether a `struct` is being used as part of a `ConstantBuffer`, or as part of a `StructuredBuffer`.  
```

Comment:
Add context to clarify whether or not this is a problem that the special attribute tries to resolve and when the problem occurs (e.g. during compilation). 
```suggestion
When AZSLc compiles a variable defined inside structs, the `[[pad_to(N)]]` attribute helps guarantee the variable's alignment, which is a problem that occurs in DX12. In DX12, for variables inside `struct` definitions, the variable's layout, offset, and size changes depending on if the struct is part of a `ConstantBuffer` or a `StructuredBuffer`.  
```

Reply:
Fixed.